src/Form/DocumentType.php

```diff
@@ -47,14 +47,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('files', FileType::class, [
                 'label' => $this->translator->trans('Files', [], 'documents'),
-                'help' => $this->translator->trans('Upload one or more files. Max file size: 10mb. File formats accepted: .pdf, .txt, .mp4, .jpeg, .png, .doc, .xls', [], 'documents'),
+                'help' => $this->translator->trans('Upload one or more files. Max file size: 80mb. File formats accepted: .pdf, .txt, .mp4, .jpeg, .png, .doc, .xls', [], 'documents'),
                 'mapped' => false,
                 'multiple' => true,
                 'constraints' => [
                     new All([
                         'constraints' => [
                             new File([
-                                'maxSize' => '1024k',
+                                'maxSize' => '80M',
```

Comment:
This should probably be configurable (the value depends on the value of `80` used in `DigitalPostHelper`).

Reply:
Made configurable with default value equivalent to total filesize allowed in digital post.